README.markdown

```diff
@@ -1,3 +1,5 @@
+Official version [underscorestring:underscore.string](https://atmospherejs.com/underscorestring/underscore.string) doesn't work from packages - see https://github.com/epeli/underscore.string/issues/419
```

Comment:
Honestly, I don't like having this on the first line in the readme. Underscore.string's primary target isn't meteor. This should be in the meteor section.